src/site/content/en/notifications/codelab-notifications-service-worker/index.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: codelab
+title: Use a Service Worker to manage notifications
+author: katejeffreys
+description: |
+  In this codelab, learn how to manage notifications with a
+  service worker.
+date: 2019-10-14
+# Add an updated date to your post if you edit in the future.
+# updated: 2019-06-27
+glitch: codelab-notifications-service-worker
+# Specify which file the glitch should start on.
+glitchPath: public/index.js  
+related_post: use-push-notifications-to-engage-users
+---
+
+In this codelab, you'll use a service worker
+to manage [notifications](/use-push-notifications-to-engage-users).
+The instructions here assume that you're already familiar with
+[service workers](https://developers.google.com/web/fundamentals/primers/service-workers)
+and the basics of requesting notification permission and sending notifications.
+If you need a refresher, see the
+[Get started with the Notifications API](/codelab-notifications-get-started)
+codelab.
```

Comment:
I added some links and an explanation of what we expect users to know when starting this post. (Flagging SWs in particular seemed important.)